docs/api-types/certificate/README.md

```diff
@@ -0,0 +1,32 @@
+# Certificates
+
+cert-manager has the concept of 'Certificates' that define a desired X.509 certificate. `Certificate` resources reference an `Issuer` or `ClusterIssuer` for information on how to obtain the certificate.
+
+A simple `Certificate` could be defined as:
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: acme-crt
+spec:
+  secretName: acme-crt
+  commonName: foo.example.com
+  dnsNames:
+  - bar.example.com  
+  issuerRef:
+    name: letsencrypt-prod
+    # We can reference ClusterIssuers by changing the kind here.
+    # The default value is Issuer (i.e. a locally namespaced Issuer)
+    kind: Issuer
+```
+
+This `Certificate` will tell cert-manager to attempt to use the referenced `letsencrypt-prod` `Issuer` resource to obtain a certificate key pair for the `foo.example.com` domain. If successful, the resulting key and certificate will be stored in a secret named `acme-crt` with keys of `tls.key` and `tls.crt` respectively. This secret will live in the same namespace as the `Certificate` resource.
```

Comment:
```
...obtain a certificate key pair for the `foo.example.com` domain...
```

Probably worth adding `and bar.example.com` here, even if it is a SAN. The note below explains why both domains end out on the certificate well 😄